.travis.yml

```diff
@@ -1,22 +1,32 @@
 language: node_js
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
This will have a significant impact on how long it takes us to get a Travis instance